### ISSUE TITLE
Time filters for memory page

### ISSUE THREAD

**Issue by pieroit** [cheshire-cat-ai/admin-vue#22]
**Is your feature request related to a problem? Please describe.**
Since we have timestamps for memories, it would be nice to have a couple of time filters in the memory page (from / to).

Steps:
1. recall endpoint ignores time filters and the time filtering is done in the admin
2. recall endpoint is able to filter by time interval directly in the backend

Filters (from/to) could be shown as `dd/mm/yyyy` and handled internally as unix timestamps (which is waaay better)

**Comment by zAlweNy26** [comment]
It would be nice to also fix per source and score.

**Comment by pieroit** [comment]
> It would be nice to also fix per source and score.

The score is easily supported (there is already a `threshold` parameter that has only to be exposed (between -1 for very dissimilar, 0 orthogonal, 1 same memory)

For the sources I guess you need an endpoint to list the unique sources? Or we insert that in the `GET collections` endpoint?

**Comment by zAlweNy26** [comment]
It would be easier with a new endpoint to list the active users

**Comment by zAlweNy26** [comment]
I would like to design this feature together if possible @pieroit , to better see where to put everything.

**Comment by pieroit** [comment]
> I would like to design this feature together if possible @pieroit , to better see where to put everything.

Yes gladly! So we can also better separate memory wipe from search ;)
